Gemfile

```diff
@@ -70,6 +71,16 @@ group :development do
   # in the background. Read more: https://github.com/rails/spring
   gem "spring"
   gem "spring-watcher-listen", "~> 2.0.0"
+
+  gem "capistrano"
+  gem "capistrano-bundler"
+  gem "capistrano-rails"
+  gem "capistrano-rbenv"
+  gem "capistrano-dotenv"
```

Comment:
__[RuboCop]__

Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `capistrano-dotenv` should appear before `capistrano-rbenv`.


<!-- Comment by [SideCI](https://sideci.com) -->